pimd/pim_tib.c

```diff
@@ -106,6 +106,11 @@ bool tib_sg_gm_join(struct pim_instance *pim, pim_sgaddr sg,
 					  __func__, result);
 			return false;
 		}
+
+		if (!pim_addr_is_any(sg.src))
```

Comment:
This function will be called for both ASM and SSM, I think there is some other issue. Why SSM has the STAR flag in the first place?

Reply:
`STAR` is used to hold coil entry with no `oif` like <*,G>. Please refer to commit 2164ed5d04db1f556e717b1f0bd3ad442c2ced2c.

Reply:
Yes so mainly the idea is when we have (*,G) and an (S,G) is created, the oil lists from (*,G) gets inherited to (S,G) but in case of SSM, we should not have any (*,G) entries, why do we have it in your case, that is my question?